iOS/DuckDuckGo/Subscription/Views/PurchaseInProgressView.swift

```diff
@@ -27,8 +27,8 @@ struct PurchaseInProgressView: View {
         static let coverOpacity = 0.6
         static let cornerRadius = 12.0
         static let shadowRadius = 10.0
-        static let lightShadowColor = Color.gray50
-        static let darkShadowColor = Color.gray95
+        static let lightShadowColor = Color(baseColor: .gray50)
+        static let darkShadowColor = Color(baseColor: .gray100)
```

Comment:
gray95 vs gray100 is intentional - have the same value in DuckUI vs DRK

https://github.com/duckduckgo/DesignResourcesKit/pull/11/files#diff-ed8a0fe2b932de59fdfa8e63bb9dc412a3849d6a5a9fd45363f21c11aa1b4262R33
https://github.com/duckduckgo/apple-browsers/blob/b38649d16322e767a2c9a66dc3d3528a5688fe1a/iOS/LocalPackages/DuckUI/Sources/DuckUI/Color.swift#L49